test/studies/lcals/LCALSMain.chpl

```diff
@@ -0,0 +1,1203 @@
+module main {
+  use FileSystem;
+  use LCALSParams;
+  use LCALSEnums;
+  use LCALSDataTypes;
+  use LCALSLoops;
+  use LCALSStatic;
+  use LCALSConfiguration;
+
+  use RunARawLoops;
+  use RunBRawLoops;
+  use RunCRawLoops;
+  use RunParallelRawLoops;
+  use RunVectorizeOnlyRawLoops;
+
+  proc allocateLoopData() {
+    const num_aligned_segments =
+      (s_loop_data.max_loop_length + 20)/LCALS_DATA_ALIGN + 1;
+    const aligned_chunksize = num_aligned_segments*LCALS_DATA_ALIGN;
+
+    writeln("\n allocateLoopData...");
+
+    for i in 0..#s_loop_data.s_num_1D_Real_arrays {
+      initData(s_loop_data.RealArray_1D[i], i+1);
+    }
+    for i in 0..#s_loop_data.s_num_1D_Nx4_Real_arrays {
+      initData(s_loop_data.RealArray_1D_Nx4[i], i+1);
+    }
+    for i in 0..#s_loop_data.s_num_1D_Indx_arrays {
+      initData(s_loop_data.IndxArray_1D[i], i+1);
+    }
+    for i in 0..#s_loop_data.s_num_1D_Complex_arrays {
+      initData(s_loop_data.ComplexArray_1D[i], i+1);
+    }
+    for i in 0..#s_loop_data.s_num_2D_Nx25_Real_arrays {
+      initData(s_loop_data.RealArray_2D_Nx25[i], i+1);
+    }
+    for i in 0..#s_loop_data.s_num_2D_7xN_Real_arrays {
+      initData(s_loop_data.RealArray_2D_7xN[i], i+1);
+    }
+    for i in 0..#s_loop_data.s_num_2D_64x64_Real_arrays {
+      initData(s_loop_data.RealArray_2D_64x64[i], i+1);
+    }
+    for i in 0..#s_loop_data.s_num_3D_2xNx4_Real_arrays {
+      initData(s_loop_data.RealArray_3D_2xNx4[i], i+1);
+    }
+    initData(s_loop_data.scalar_Real, 21);
+    // ?? s_loop_data->RealArray_scalars;
```

Comment:
It was something I wasn't sure how best to represent when first porting it.  Looks like I got it working and forgot to remove the ?? comment.  Deleted.